net-wireless/yatebts/files/yatebts-5.0.0-gcc6.patch

```diff
@@ -0,0 +1,31 @@
+--- a/mbts/GPRS/MSInfo.cpp
++++ b/mbts/GPRS/MSInfo.cpp
+@@ -638,7 +638,7 @@
+         if (msPCHDowns.size() > 1) {
+             std::ostringstream os;
+             msDumpChannels(os);
+-            GPRSLOG(INFO,GPRS_MSG|GPRS_CHECK_OK) << "Multislot assignment for "<<this<<os;
++            GPRSLOG(INFO,GPRS_MSG|GPRS_CHECK_OK) << "Multislot assignment for "<<this<<(!os.fail());
```

Comment:
you could've also `static_cast<bool>`'d it, same semantics, imo a bit easier to read, due to no negative logic